lasio/writer.py

```diff
@@ -73,15 +80,17 @@ def write(
         las.version["WRAP"] = HeaderItem("WRAP", "", "NO", "One line per depth step")
     lines = []
 
+    versionSectionToWrite = deepcopy(las.version)
```

Comment:
Change camel case to underscore separated lower case, to keep styles consistent

Reply:
Commit d427d2c, changes the versionSectionToWrite variable name  to version_section_to_write. 

Thanks,
DC